src/vs/github1s/notification.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @file github1s notification
+ * @author netcon
+ */
+
+import 'vs/css!./notification';
+
+const NOTIFICATION_STORAGE_KEY = 'GITHUB1S_NOTIFICATION';
+// change it if a new notification should be shown
+const NOTIFICATION_STORAGE_VALUE = '20210212';
+
+const notification = {
+	title: 'ATTENTION: This page is NOT officially provided by GitHub.',
+	content: 'GitHub1s is open source project, not officially provided by GitHub.',
+	link: 'https://github.com/conwnet/github1s',
```

Comment:
there is the notification message

Reply:
Suggestion for content: `GitHub1s is an open source project, which is not officially provided by GitHub.`

Reply:
Thanks~